internal/services/oracle/oracle_autonomous_database_backup_resource_test.go

```diff
@@ -0,0 +1,246 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: MPL-2.0
+
+package oracle_test
+
+import (
+	"context"
+	"fmt"
+	"strings"
+	"testing"
+
+	"github.com/hashicorp/go-azure-helpers/lang/pointer"
+	"github.com/hashicorp/go-azure-sdk/resource-manager/oracledatabase/2025-03-01/autonomousdatabasebackups"
+	"github.com/hashicorp/go-azure-sdk/resource-manager/oracledatabase/2025-03-01/autonomousdatabases"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/acceptance"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/acceptance/check"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/clients"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/services/oracle"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/tf/pluginsdk"
+)
+
+type AutonomousDatabaseBackupResource struct{}
+
+func TestAutonomousDatabaseBackupResource_basic(t *testing.T) {
+	data := acceptance.BuildTestData(t, oracle.AutonomousDatabaseBackupResource{}.ResourceType(), "test")
+	r := AutonomousDatabaseBackupResource{}
+	data.ResourceTest(t, r, []acceptance.TestStep{
```

Comment:
All of the tests are currently failing because the resource currently isn't supported in the default `westeurope` region, can we add this?

```suggestion
	// TODO - Remove this override when resource is rolled out to westeurope - currently only supported in limited regions
	data.Locations.Primary = "eastus2"
	
	data.ResourceTest(t, r, []acceptance.TestStep{
```